examples/always_returning_example.py

```diff
@@ -0,0 +1,21 @@
+def loop():
+    for j in range(0, 1):
```

Comment:
There should be an error message in both.

But it appears you didn't understand what I asked you to do: change `range(0, 1)` to something like `range(0, 10)`. `range(0, 1)` is an error in itself that we certainly do not want to model, and perhaps we should check for.